packages/backend/src/workers/provider/LlamaCppPython.ts

```diff
@@ -56,6 +58,18 @@ export class LlamaCppPython extends InferenceProvider {
 
     const envs: string[] = [`MODEL_PATH=/models/${modelInfo.file.file}`, 'HOST=0.0.0.0', 'PORT=8000'];
     envs.push(...getModelPropertiesForEnvironment(modelInfo));
+    if (isMacArm()) {
```

Comment:
Following https://github.com/containers/podman-desktop-extension-ai-lab/pull/1339 we might want to have something like 

```
const vmType = await this.podmanConnection.getVMType();
if(vmType  === VMType.LIBKRUN) ...
```